src/components/products-card/index.ts

```diff
@@ -0,0 +1 @@
+export { ProductsCard } from './products-card';
```

Comment:
rm it

Reply:
<img width="355" alt="Screenshot 2024-05-03 at 12 17 20" src="https://github.com/greyzerua/fe-react-2024/assets/118940133/b173d064-0d6a-4158-b64b-95c732fffbac">

Reply:
fixed